data/human/navy missions intro.txt

```diff
@@ -0,0 +1,1496 @@
+			`	"That makes sense," you remark, turning towards the admiral. "If the pirates decide to attack again, we might not be able to drive them off with what we have."`
+			`	"Precisely. Now, Lieutenant <last>, if you would, please follow me to my quarters here."`
+			`The admiral's quarters turns out to be a small grouping of construction trailers connected to each other. "Shut the door, please," he says as you enter the office. As Danforth takes a seat behind his desk, you glance about the room. The walls have tables, filing cabinets, and a few calendars with target objectives to be met.`
+				goto debriefing
+			label youDoneGoofed
+			apply
+				payment -1000000
+			`As you make your way through the "shipyard" (admittedly, more like a clearing with gun emplacements) to Admiral Danforth, you notice the auxiliaries are being modified - their rear halves are still the giant cargo containers, but the front of each auxiliary appears to have gained a set of fins - and a sleeker body. "They're being retrofitted to carry more personnel," Danforth says as he suddenly appears by your side. "For the next stage of the reconstruction, we need to bring in personnel and equipment - in addition to fresh recruits, we also need better defenses than the turrets we currently have."`
+			`	"That makes sense," you remark, turning towards the admiral. "If the pirates decide to attack again, we might not be able to drive them off with what we have."`
+			`	"Precisely. Now, Lieutenant <last>, if you would, please follow me to my quarters here."`
+			`The admiral's quarters turns out to be a small grouping of construction trailers connected to each other. "Shut the door, please," he says as you enter the office. As Danforth takes a seat behind his desk, you glance about the room. The walls have tables, filing cabinets, and a few calendars with target objectives to be met.`
+			`	"I understand there was...an incident...on Lodestone. Is that accurate?" Before you can even reply, he shows you a video recording from the docks where you almost got in a fight. Watching in wide-eyed horror as your actions are displayed for all to see, you feel the temperature in the room drop sharply.`
+			`"In case you're wondering, the dockworkers sent me that recording."`
+			`	"Sir, I-"`
+			`	"Don't," he harshly cuts you off, standing up. "I don't need to hear your excuses, <last> - your conduct speaks volumes more than your words. And for every action...there are consequences." Suddenly, you receive an email from your bank account - 1 million credits were just returned to the Navy. In the brief moments you look at your communicator's display, Danforth is already uncomfortably close to you.`
+			`	"Rather than revoke your salary, I opted for something a bit more...immediate," he explains, calmly. "As I gave you the 1.5 million, I can just as easily take it away."`
```

Comment:
The ramifications of this for how the banking establishment in the republic works are frankly dystopic and terrifying.